src/main/java/edu/wpi/first/pathweaver/PathDisplayController.java

```diff
@@ -198,8 +201,8 @@ public void setupWaypoint(Waypoint waypoint) {
   @FXML
   private void keyPressed(KeyEvent event) {
     if ((event.getCode() == KeyCode.DELETE || event.getCode() == KeyCode.BACK_SPACE)
-        && isDeletable(selectedWaypoint)) {
-      delete(selectedWaypoint);
+        && isDeletable(getSelectedWaypoint())) {
+      delete(getSelectedWaypoint());
```

Comment:
Should call `selectedWaypoint.set(null)` after the delete.
I believe it should also be called in the `currentPath.addListener` lambda on line 87 so that selecting a different path will clear waypoint selection